src/main/java/core/basesyntax/RandoM.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class RandoM extends Enum {
+    public Color getRandomColor() {
+        Random random = new Random();
+        int index = random.nextInt(Color.values().length);
+        return Color.values()[index];
+    }
+
+    public int getRandomNumber() {
+        Random random = new Random();
+        return random.nextInt(100);
```

Comment:
The number '100' is a magic number in this context. It should be replaced with a named constant to improve code readability and maintainability.